packages/manager/modules/catalog-price/src/catalog-price.html

```diff
@@ -1,9 +1,61 @@
-<span data-ng-if="::$ctrl.price">
-    <span data-ng-if="::$ctrl.isAsiaFormat()">
-        <strong data-ng-class="{ 'd-block': $ctrl.block }">
+<div data-ng-if="!$ctrl.cdnCase">
```

Comment:
This is too specific to a product and prevents component reusability

Reply:
yes it's part of changes i plan to do with Cyrille

Reply:
fixed